packages/webapp/src/views/land/check-ownership-proof-file.html

```diff
@@ -15,6 +15,7 @@
         {% set name = 'checkLandOwnership' %}
         {% set id = 'check-upload-correct-yes' %}
         {% set downloadUrl = 'download-land-ownership-file' %}
+        {% set checkUploadCorrectNoText = 'No, upload a different file' %}
```

Comment:
is this right? my understanding is yes and no part of html is extracted out to a common file in line 19.

I have already updated common file in my PR which will apply in all places once merged to master. 

please check

Reply:
Is this ok to be resolved?